plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -121,6 +131,13 @@ public static List<String> getPartitioning(Map<String, Object> tableProperties)
         return partitioning == null ? ImmutableList.of() : ImmutableList.copyOf(partitioning);
     }
 
+    @SuppressWarnings("unchecked")
+    public static List<String> getSortOrder(Map<String, Object> tableProperties)
```

Comment:
The sort-order is stored to metadata, but it isn't used from anywhere if my understanding is correct. What's the benefit adding this property? Don't we need to respect the property during writes? It would be nice if you can add tests showing the benefit.

Reply:
@ebyhr this PR is the first step in supporting `sorted_by’.

So, this PR is only intended to add the support of `sorted_by` for the `CREATE TABLE ` DDL syntax.

subsequently we can add for `ALTER TABLE` and then to support while writing. 

DDL changes will help when Spark is being used for the ingestion (and that is the case we see it almost all the time). Spark uses icebergs sorting spec to write as well.

Reply:
I disagree adding `sorted_by` property without proper support. It's confusing to users.